src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -33,17 +36,24 @@ public AddCommand parse(String args) throws ParseException {
 
         boolean isRemindPresent = args.matches(".*\\bremind\\b$");
         boolean isRemindTypo = false;
+        boolean isPriorityPresent = true;
 
         if (!isRemindPresent) {
             isRemindTypo = args.matches(".*rem[a-z]*$");
         }
 
+        if (ArgumentTokenizer.findPrefixPosition(args, PREFIX_PRIORITY.getPrefix(), 0) == NO_OCCURRENCE_FOUND) {
+            isPriorityPresent = false;
+        }
+
         if (isRemindPresent || isRemindTypo) {
+            // remove remind from args as ArgumentTokenizer cannot parse remind without prefix
             String argsWithoutRemind = args.replace(" remind", "");
             argMultimap = ArgumentTokenizer.tokenize(argsWithoutRemind, PREFIX_NAME, PREFIX_DEADLINE,
-                    PREFIX_MODULE_CODE);
+                    PREFIX_MODULE_CODE, PREFIX_PRIORITY);
```

Comment:
What if the user inputs remind without a priority?

Reply:
Nothing will happen! The argMultimap will just parse those that are present into the HashMap